recipes-multimedia/ffmpeg/ffmpeg_4.4.1.bb

```diff
@@ -183,3 +183,5 @@ INSANE_SKIP:${MLPREFIX}libpostproc = "textrel"
 # Downgrade for NXP BSP
 DEFAULT_PREFERENCE = "-1"
 COMPATIBLE_MACHINE = "(imx-nxp-bsp)"
+
+LICENSE_FLAGS_ACCEPTED += " commercial "
```

Comment:
This should be removed, it was a mistake

Reply:
Removed. Should I also remove it from `gstreamer1.0-plugins-ugly`?

Reply:
Yes, I didn't notice that one.

Reply:
Ok, removed, thank you